images/aws-load-balancer-controller/tests/main.tf

```
@@ -13,6 +13,9 @@ data "oci_string" "ref" { input = var.digest }
 
 resource "random_pet" "suffix" {}
```

This could be a part of the commented block.

```suggestion
// resource "random_pet" "suffix" {}
```